test/test_unary_ufuncs.py

```diff
@@ -490,6 +493,24 @@ def test_sqrt_complex_edge_values(self, device, dtype):
         x = torch.tensor(-1.0000e+20 - 4988429.2000j, dtype=dtype, device=device)
         self.compare_with_numpy(torch.sqrt, np.sqrt, x)
 
+    @unittest.skipIf(not TEST_SCIPY, "Requires SciPy")
+    @dtypes(torch.float, torch.double)
+    def test_digamma_special(self, device, dtype):
```

Comment:
Now that digamma is an OpInfo and has this special test, `test_digamma` in test_torch.py should be moved here or deleted. `digamma` also has an entry in `tensor_op_tests` in test_torch.py that can be removed. And it has a `TorchMathTestMeta` that can be removed.

Reply:
While moving `test_digamma` to `test_unary_ufuncs`.
I found that there is divergence in digamma between SciPy and PyTorch
```python
>>> import torch
>>> import scipy.special
>>> x = torch.tensor(-1931.99999994)
>>> x = torch.tensor(-1931.99999994, dtype=torch.double)
>>> x.digamma()
tensor(-16666631.6273, dtype=torch.float64)
>>> scipy.special.digamma(x.numpy())
-16666645.861379756
```

I also verified the value from Cephes which is more or less closer (though not exactly) to SciPy,
![Screenshot from 2020-11-28 12-51-54](https://user-images.githubusercontent.com/19503980/100496262-9f193d00-3178-11eb-9872-9f829cc22f5d.png)

Should it be fixed in this PR (should it be in scope of this PR) or make an issue to follow it up after this?

Reply:
Thank you for being so thorough. Please file a separate issue, cite it as a comment above the digamma OpInfo, and let's follow-up in a separate PR.